go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/julienschmidt/httprouter v1.2.0
 	github.com/pkg/errors v0.9.1 // indirect
-	github.com/stretchr/testify v1.4.0 // indirect
+	github.com/stretchr/testify v1.4.0
```

Comment:
I fail to see why this line changed... We are not using `testify` directly, are we?

Reply:
Must have been from a previous run. Fixed by running `go mod tidy`.

`/tools/prepare-source.sh` should have found this, and it did, but it's hooked up wrong so we didn't notice. Filed https://github.com/osbuild/osbuild-composer/pull/307 to fix that.